README.md

```diff
@@ -1 +1,181 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="codeeq.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hellow Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello Microverse Project <a name="about-project"></a>
+
+This is my First Microverse Project. I created it using HTML and CSS. In addition, i added Linters for Html and Css.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-HTML
+2-CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add index.html file.
+- Add style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://codeeq-verse.netlify.app/">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+1-Web Browser
+2-Code Editor
+3-Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this Command:
+
+```sh
+  cd my-folder
+  git clone git@github.com:GhausAliKhan/Hello-Microverse-Project.git
+```
+
+### Install
+
+First you need to install the node package files
+Run this command:
+
+```sh
+  cd Hello-Microverse-Project
+  npm install
+```
+
+### Usage
+
+- In order to use this project you either create branch to add more features.
+- But in order to run the project open index.html with live server
+
+### Run Linters
+
+I have added the linters or tests in order to avoid anykind of typos and ensure the good practices in this project. So while using this repository if you need to check your work in html and css you can use below commands
+
+- in order to run the linter test on index.html:
+
+```sh
+ npx hint .
+```
+
+- in order to run the linter test on stylesheets:
+
+```sh
+ npx styleint "**/*.{css,scss}"
+```
+
+### Deployment
+
+- You can easily use github pages to deploy your project
+  <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pagess">Useful link about Github Pages</a>
```

Comment:
- Hey, let's also update this link as the link is broken. You can tell users the platforms they can use to deploy the app, e.g.: Github pages, netlify, etc. Let's update this to make the readme more professional.

Reply:
This is also updated with working links.

Reply:
@zieeco Do i need to re-submit this PR link for re-review or not?